pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -137,7 +138,15 @@ func Execute(configFile io.Reader) {
 	app.Config.HTTP.Headers.Set("X-Registry-Supports-Signatures", "1")
 
 	app.RegisterHealthChecks()
-	handler := alive("/", app)
+	handler := http.Handler(app)
```

Comment:
> we already have a crude pullthrough limiter that limits max simultaneous writes.
It's better to think that we haven't, it has serious leaking problems (and it's not a pull-through limiter, but a storage writer limiter).

> If pull though proxy needs to be limited
Pull-through can trigger mirroring which uses blob writers which consume memory. So... I don't know, maybe we needn't. Do we want to have the same limiter for mirroring and for PUT/PATCH requests?